configure.ac

```diff
@@ -793,6 +793,9 @@ if test "$cross_compiling" = yes; then
 	*-*-vxworks*)
 		_host_ident=$host_cpu
 		;;
+  *-*-emscripten)
+    _host_ident=$(cat $(dirname $(which emcc))/emscripten-version.txt | cut -d- -f1)-$host_cpu
```

Comment:
```suggestion
    _host_ident=$(emcc -dumpversion)-$host_cpu
```

Reply:
Good to know about this one, it does what I was hoping `--version` would do.